src/controller/java/BUILD.gn

```diff
@@ -122,12 +114,28 @@ android_library("java") {
     "zap-generated/chip/devicecontroller/ClusterWriteMapping.java",
   ]
 
+  if (build_java_matter_controller) {
+    deps = [
+      "${chip_root}/examples/java-matter-controller/stub_src",
```

Comment:
We should move the stub from java-matter-controller example app to sdk so that any Java apps can use it. 
deps = [
    "${chip_root}/src/controller/java/src/stub",

Reply:
there is no java app that needs this stub, so i prefer putting here, this code would never be expected to appear in core.

Reply:
The main concern I have is not about if we have any other Java app use this, my concern is if we can make make core SDK gn reverse depends on an component defined in Example gn file.

Reply:
just move it to third_party/java_deps, feel it is more natural to put this dummy stub there., thanks